packages/server/test/unit/specs_spec.coffee

```diff
@@ -56,23 +57,48 @@ describe "lib/util/specs", ->
         expect(files.length).to.equal(1)
         expect(files[0].name).to.equal("coffee_spec.coffee")
 
+    it "uses glob to process config.testFiles", ->
+      config.get(FixturesHelper.projectPath("various-file-types"))
+      .then (cfg) ->
+        cfg.testFiles = "{coffee_*.coffee,js_spec.js}"
+        specsUtil.find(cfg)
+      .then (files) ->
+        debug("found spec files %o", files)
+        expect(files.length).to.equal(2)
+        expect(files[0].name).to.equal("coffee_spec.coffee")
+        expect(files[1].name).to.equal("js_spec.js")
+
+    it "allows array in config.testFiles", ->
+      config.get(FixturesHelper.projectPath("various-file-types"))
+      .then (cfg) ->
+        cfg.testFiles = ["coffee_*.coffee", "js_spec.js"]
+        specsUtil.find(cfg)
+      .then (files) ->
+        debug("found spec files %o", files)
+        expect(files.length).to.equal(2)
+        expect(files[0].name).to.equal("coffee_spec.coffee")
+        expect(files[1].name).to.equal("js_spec.js")
+
     it "filters using specPattern", ->
       config.get(FixturesHelper.projectPath("various-file-types"))
       .then (cfg) ->
-        specsUtil.find(cfg, [
+        specPattern = [
           path.join(cfg.projectRoot, "cypress", "integration", "js_spec.js")
-        ])
+        ]
+        specsUtil.find(cfg, specPattern)
       .then (files) ->
         expect(files.length).to.equal(1)
         expect(files[0].name).to.equal("js_spec.js")
 
     it "filters using specPattern as array of glob patterns", ->
       config.get(FixturesHelper.projectPath("various-file-types"))
       .then (cfg) ->
-        specsUtil.find(cfg, [
+        debug("test config testFiles is %o", cfg.testFiles)
```

Comment:
adding a few debug statements to the test itself is useful when iterating on the test and understanding what is going on